Modules/_tkinter.c

```diff
@@ -3399,7 +3407,7 @@ static struct PyModuleDef _tkintermodule = {
     NULL,
     NULL,
     NULL,
-    NULL
+    module_free
```

Comment:
While we are here, please use syntax like `.m_name = ` for struct initialization.

I think that you need to define not only m_free, but also m_clear and m_traverse.

Reply:
Will get to that a little later today. Is the convention to use `.m_whatever = NULL` for unused fields as well?

Reply:
You can omit members if value is NULL.

Reply:
All fixed.